front-end/src/reducers/pages.js

```diff
@@ -1,9 +1,30 @@
 const defaultState = {
-
+  storiesList: {
+    searchResults: []
+  }
 }
 
 const pages = (state = defaultState, action) => {
-  return state;
+  switch (action.type) {
+    case 'STORIES_SEARCH_SUCCESS':
+      return {
+        ...state,
+        storiesList: {
+            ...state.storiesList,
+            searchResult: action.response
```

Comment:
You're only interested in the result. The entities are already stored by the `entities` reducer.